core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/histogram/HistogramChartOpDesc.scala

```diff
@@ -0,0 +1,102 @@
+package edu.uci.ics.texera.workflow.operators.visualization.histogram
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType, Schema}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+class HistogramChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+  @JsonProperty(required = false, defaultValue = "Histogram")
+  @JsonSchemaTitle("Title")
+  @JsonPropertyDescription("Add a title to your histogram plot.")
+  var title: String = "Histogram"
+
+  @JsonProperty(value = "value", required = true)
+  @JsonSchemaTitle("Value Column")
+  @JsonPropertyDescription("Column for analyzing")
+  @AutofillAttributeName
+  var value: String = ""
+
+  @JsonProperty(required = false)
+  @JsonSchemaTitle("Category Column")
+  @JsonPropertyDescription("Column for assigning different colors")
+  @AutofillAttributeName
+  var category: String = ""
+
+  @JsonProperty(defaultValue = "false", required = false)
+  @JsonSchemaTitle("Separate Graphs")
```

Comment:
Make this name more intuitive. The title does not reflect that it is for different categories.

Reply:
I rename the parameter. If it still not intuitive enough please let me know.